File: LeetCode-Programs/Find All Numbers Disappeared in an Array.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        num=set(nums)
        new_list=[]
        for i in range(1,len(nums)+1):
            if not i in num:
                new_list.append(i)
        return new_list

File: LeetCode-Programs/Find All Numbers Disappeared in an Array.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        set<int> s(nums.begin(),nums.end());        
        vector<int> vec;
        for(int i=1;i<=nums.size();i++)
        {
            if(s.count(i)==0)              
                vec.push_back(i);
        }
        return vec;
    }
};
